bootstrap-compiler

```diff
@@ -6,7 +6,7 @@ const fs = require("fs");
 (async function() {
   console.log("pwd=" + process.cwd());
   let files = await fs.promises.readdir(".");
-  console.log("files=" + files.join("\n  "));
+  console.log("files=\n  " + files.join("\n  "));
```

Comment:
all of this could be removed, it was only used for debugging a workflow problem and I should have removed it before